### PR TITLE
1.0.lerna

### PR DESCRIPTION
**Pull request by lucienbertin**
updated lerna.json and other stuff so lerna can discover `@lucca-front/ng` which is under `/packages/ng/dist` and dont publish `ng-internal` which used to be `/packages/ng`

### CODE REVIEW

File: packages/ng/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "type": "git",
     "url": "git+https://github.com/LuccaSA/lucca-front.git"
   },
+  "private": true,
```
Comment (jefBinomed) [inline packages/ng/package.json]: Why Private ? Lucca-Front isn't supposed to be public ?

Comment (lucienbertin) [reply packages/ng/package.json]: its not this package.json that gets published, this one serves as an entry point to build /librairies, then we publish whats inside /dist

Comment (lucienbertin) [reply packages/ng/package.json]: run `yarn build` then look whats inside `/dist`

Comment (lucienbertin) [reply packages/ng/package.json]: @jefBinomed ??

Comment (jefBinomed) [reply packages/ng/package.json]: nothing more ;) I didn't reopen the PR during the whole day

**Pull request merged**